airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -700,7 +700,7 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
-- dockerImage: "airbyte/source-amazon-ads:0.1.27"
+- dockerImage: "airbyte/source-amazon-ads:0.1.28"
```

Comment:
you don't need to bump version here, it's done automatically by publishing scripts